config/configure_options/private_configure_options/xeon_biolaptop_mpi

```diff
@@ -124,15 +124,15 @@ LD=mpif77
 #CXXFLAGS="-O3 -Wall -fPIC"
 #CFLAGS="-O3 -fPIC"
 #FFLAGS="-O3 -fPIC"
-#--with-blacs=/home/mheil/local/mumps_and_scalapack_default_installation/lib
-#--with-scalapack=/home/mheil/local/mumps_and_scalapack_default_installation/lib/libscalapack.a
-#--with-pord=/home/mheil/local/mumps_and_scalapack_default_installation/lib/libpord.a 
-#--with-mumps=/home/mheil/local/mumps_and_scalapack_default_installation
-#--with-blas=/home/mheil/local/lib/blas/blas.a
-#--with-lapack=/home/mheil/local/lib/lapack/lapack.a
-#--with-hypre=/home/mheil/local/hypre_default_installation_mpi
-#--with-trilinos=/home/mheil/local/trilinos_default_installation_mpi
+#--with-blacs=/home/matthias/local/mumps_and_scalapack_default_installation/lib
+#--with-scalapack=/home/matthias/local/mumps_and_scalapack_default_installation/lib/libscalapack.a
+#--with-pord=/home/matthias/local/mumps_and_scalapack_default_installation/lib/libpord.a 
+#--with-mumps=/home/matthias/local/mumps_and_scalapack_default_installation
+#--with-blas=/home/matthias/local/lib/blas/blas.a
+#--with-lapack=/home/matthias/local/lib/lapack/lapack.a
+#--with-hypre=/home/matthias/local/hypre_default_installation_mpi
+#--with-trilinos=/home/matthias/local/trilinos_default_installation_mpi
```

Comment:
Is there a better way of handling the configure options rather than adding private configure options to the public repo?